api/app/Models/PoolCandidate.php

```diff
@@ -250,9 +252,9 @@ public static function scopeExpiryStatus(Builder $query, ?string $expiryStatus)
         return $query;
     }
 
-    public function scopeNotDraft(Builder $query): Builder
+    public function scopeNotDraft(Builder $query, $where, bool $notDraft = true): Builder
     {
-        return $query->whereNotIn('pool_candidate_status', ['DRAFT', 'DRAFT_EXPIRED']);
+        return $notDraft ?  $query->whereNotIn('pool_candidate_status', ['DRAFT', 'DRAFT_EXPIRED']) : $query;
```

Comment:
We should ensure the logic for `scopeNotDraft` and `isDraft` matches, by updating this to check submitted_at.

Reply:
Updated in [update draft logic to only check submitted_at](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/0cea9ded0671ce3f470d698d4493eef8bb6830e3)